FinancialTracker/ViewControllers/TabBarController.swift

```diff
@@ -93,20 +94,17 @@ class TabBarController: UITabBarController {
 
         for viewController in viewControllers {
             if let navigationController = viewController as? UINavigationController {
-                let signOutButton = UIBarButtonItem(title: "Sign Out", style: .plain, target: self, action: #selector(signOut))
                 switch navigationController.topViewController {
                 case let homeVC as HomeViewController:
                     homeVC.authManager = authManager
-                    homeVC.navigationItem.rightBarButtonItem = signOutButton
                 case let profileVC as ProfileViewController:
                     profileVC.authManager = authManager
+                    let signOutButton = UIBarButtonItem(title: "Sign Out", style: .plain, target: self, action: #selector(signOut))
```

Comment:
move the UI configuration to the `ProfileViewController` itself, in the `viewDidLoad` for example?